app/views/subscribers/_form.html.haml

```diff
@@ -53,4 +53,4 @@
       = f.submit nil, class: "btn-primary"
       = link_to 'Cancel', subscribers_path, class: 'btn-link'
       - if @editing
-        = link_to "Delete", subscriber, method: :delete, data: { confirm: "Are you sure you want to delete the location '#{subscriber.name}'" }, class: 'btn-secondary pull-right'
+        = link_to "Delete", subscriber, method: :delete, data: { confirm: "You're about to permanently delete this subscriber. This action CANNOT be undone. Are you sure you want to proceed?" }, class: 'btn-secondary pull-right'
```

Comment:
I bet all delete links could be generalized to a helper having only to specify the entity to delete